git-extra/git-self-upgrade

```diff
@@ -237,7 +237,7 @@ git_update () {
 			esac
 		elif test -n "$use_gui"
 		then
-			git gui--askyesno --title "Git Update Available" \
+			git gui--askyesno --title "Git Upgrade Available" \
```

Comment:
I am not entirely sure if we should adjust the sentences here. I think the answer is yes, but this is a bit inconsistent with the terminology here: https://github.com/larsxschneider/build-extra/blob/9ede042be813cce30ff4101c45772ee6b6ac9ca0/installer/install.iss#L1737-L1740